protostar/configuration_file/configuration_file.py

```diff
@@ -52,9 +41,12 @@ def get_command_argument(
     ]:
         ...
 
-    def create_configuration_file_data(
+    def create_model(
```

Comment:
```suggestion
    def read(
```

given that there is `save` function ?

also from fmt perspective, this could be a one liner, but the comma forces Black to put new line style here